test/common/config.h

```diff
@@ -74,4 +74,10 @@ const unsigned MByte = 1024*1024;
 #define __TBB_TEST_SKIP_AFFINITY 1
 #endif
 
+#if defined(__has_feature)
+#if __has_feature(address_sanitizer)
+    #define __TBB_TEST_USE_ADDRESS_SANITIZER 1
```

Comment:
It seems `__has_feature` is the Clang-only extension and some alternative way is needed here is needed to support GCC as well

Reply:
`__SANITIZE_ADDRESS__`
https://gcc.gnu.org/onlinedocs/gcc-4.8.0/cpp/Common-Predefined-Macros.html